RWFramework/RWFramework/Playlist/AudioTrack.swift

```diff
@@ -102,6 +102,16 @@ extension AudioTrack {
         }
     }
     
+    /// Returns currently playing asset
+    func getCurrentAsset() -> Asset? {
```

Comment:
I would generally avoid getters whenever humanly possible (they are the devil), especially given Swift's dynamic properties. So, just make `AudioTrack.currentAsset` public.